src/cfnlint/config.py

```diff
@@ -33,15 +33,17 @@
 LOGGER = logging.getLogger('cfnlint')
 
 
-def configure_logging(debug_logging):
+def configure_logging(debug_logging, info_logging):
     """Setup Logging"""
     ch = logging.StreamHandler()
     ch.setLevel(logging.DEBUG)
 
     if debug_logging:
         LOGGER.setLevel(logging.DEBUG)
-    else:
+    elif info_logging:
```

Comment:
As written it seems like info logging would win. Perhaps reverse so highest level wins?